### ISSUE TITLE
Cleanliness of Code

### ISSUE THREAD

**Issue by cvodehnal** [TheDanAlatriste/web-iv-portfolio-site#7]
It looks like you ran your index and project page through an html formatter, looking very good. but it looks like you have some extra returns in your css.
EX. 
# project-images img:hover {

```
-webkit-filter: grayscale(0%);
```

}
# more-projects img {

```
-webkit-filter: grayscale(100%); /* Chrome 19+ & Safari 6+ */
```

}

just has a lot of spacing to it, and wasted space.

**Comment by cvodehnal** [comment]
I'd say Proficient on this one=D

**Comment by e-m-durham** [comment]
Yeah, you just need to make sure you have consistent returns and spacing in CSS, but it's clear you made an effort to organize. Also, group selectors where it makes sense especially if the properties are identical.

```
    #about {
        padding: 30px 5%;
    }

    #contact {
        padding: 30px 5%;
    }
```

can become

```
    #about, #contact {
        padding: 30px 5%;
    }
```

Try to make sure you are using shorthand when you can

```
    .project-description {
        padding-top:1em;
        padding-left:15px;
        padding-right:15px;
    }
```

becomes

```
    .project-description {
        padding-top:1em 15px ? 15px;
    }
```

Also, no need to define larger column widths the same as their smaller column counterpart if they are equal. Foundation is mobile first and will inherit up. Line 30 of project-name.html `class="column large-12 medium-12"` can be `class="column medium-12"`

**Comment by e-m-durham** [comment]
Don't uppercase text in your html, you can use CSS to control that, uppercased text might change the context or meaning of your copy on a screenreader and that might not be your intention.
